_episodes/04-drawing-bitwise.md

```diff
@@ -45,103 +46,90 @@ upper-left coordinate of the sub-area we are interested in is *(44, 357)*,
 while the lower-right coordinate is *(720, 740)*. These coordinates are shown
 in *(x, y)* order. 
 
-A Python program to create a mask to select only that area of the image would start
+Python code to create a mask to select only that area of the image would start
 with a now-familiar section of code to open and display the original image. (Note
 that the display portion is used here for pedagogical purposes; it would probably
 not be used in production code.)
 
 ~~~
 '''
- * Python program to use OpenCV drawing tools to create a mask.
+ * Python program to use skimage drawing tools to create a mask.
  *
 '''
-import cv2
+import skimage
+from skimage.viewer import ImageViewer
 import numpy as np
 
 # Load and display the original image
-image = cv2.imread(filename = "maize-roots.tif")
-cv2.namedWindow(winname = "original", flags = cv2.WINDOW_NORMAL)
-cv2.imshow(winname = "original", mat = image)
-cv2.waitKey(delay = 0)
+image = skimage.io.imread("maize-roots.tif")
+viewer = ImageViewer(image)
+viewer.show()
```

Comment:
in the `cv2` version the window was given a window name. With `skimage` one would have to use the `setWindowTitle()` method (from `PyQt5.QtWidgets...`)

Reply:
I have omitted window titles in the other example for now.